app-nexus-network/app-nexus-network-htb.js

```diff
@@ -184,6 +184,9 @@ function AppNexusNetworkHtb(configs) {
                 consent_required: gdprStatus.applies,
                 consent_string: gdprStatus.consentString
             };
+
+            var uspStatus = ComplianceService.usp.getConsent();
+            queryObj.us_privacy = uspStatus.consentString;
```

Comment:
The correct property name for usp consent string is uspString.
Please change to upsString.